.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. Make sure to read the Contributing Guide before submitting your PR: https://github.com/helm/helm/blob/main/CONTRIBUTING.md
```

Comment:
Can you please me sure it points to the correct contributing guide. Notice, the URL points to helm/helm.